components/developer/clang-13/Makefile

```diff
@@ -0,0 +1,110 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Aurelien Larcher.  All rights reserved.
+# Copyright 2019 Alexander Pyhalov
+# Copyright 2019 Michal Nowak
+# Copyright 2019 Till Wegmüller
+# Copyright 2022 Klaus Ziegler
+#
+
+BUILD_BITS=		64
+BUILD_STYLE=		cmake
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		clang-13
+LLVM_NAME=		llvm
+CLANG_NAME=		clang
+
+COMPONENT_VERSION=	13.0.0
+COMPONENT_MAJOR_VERSION= \
+	$(shell echo $(COMPONENT_VERSION) | $(GSED) -e 's/\([0-9]\+\.[0-9]\+\).*/\1/')
+COMPONENT_PROJECT_URL=	http://llvm.org/
+COMPONENT_FMRI=		developer/clang-13
+COMPONENT_CLASSIFICATION=	Development/C
+COMPONENT_SUMMARY=	LLVM tools and Clang compiler
+COMPONENT_SRC=		$(LLVM_NAME)-$(COMPONENT_VERSION).src
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH= sha256:408d11708643ea826f519ff79761fcdfc12d641a2510229eec459e72f8163020
+COMPONENT_ARCHIVE_URL=  https://github.com/llvm/llvm-project/releases/download/llvmorg-$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE=	BSD
+COMPONENT_LICENSE_FILE=	LICENSE.TXT
+
+COMPONENT_ARCHIVE_1=    $(CLANG_NAME)-$(COMPONENT_VERSION).src.tar.xz
+COMPONENT_ARCHIVE_HASH_1= sha256:5d611cbb06cfb6626be46eb2f23d003b2b80f40182898daa54b1c4e8b5b9e17e
+COMPONENT_ARCHIVE_URL_1=  https://github.com/llvm/llvm-project/releases/download/llvmorg-$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE_1)
+
+PYTHON_VERSION = 3.9
+
+COMPONENT_BUILD_ARGS=	-j12
+
```

Comment:
Please remove COMPONENT_BUILD_ARGS here. It is set by the environment to an appropriate value and needs only be changed when problems occur.